testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -501,14 +554,47 @@ impl EventDispatcher {
                         );
                     }
                 }
-                events.push((!receipt.post_condition_aborted, tx_hash, event));
+                events.push((!receipt.post_condition_aborted, tx_hash, event.clone()));
```

Comment:
Is it possible to avoid this clone? These events could get large memory-wise, and this clone would end up magnifying the memory consumption of the event dispatcher.